File: ts-emit-dest/src/app/MyService/AdvancedMessage2.ts
import {SubMessage as $SubService_SubService_SubMessage} from "..//SubService/SubService/SubMessage";
export class AdvancedMessage2 : MyService.SubService.SubService.SubMessage<string>
{
    public override string __reflection { get; set; } = "MyService.AdvancedMessage2";
    public string story { get; set; }
}

File: pywebsocket/MyService/AdvancedMessage2.py
from __future__ import annotations
from MyService.SubService.SubService.SubMessage import SubMessage as ImportedRef__MyService_SubService_SubService_SubMessage
from typing import TypedDict

class AdvancedMessage2(ImportedRef__MyService_SubService_SubService_SubMessage[str]):
    def __init__(self):
        super().__init__()
        self.__reflection = 'MyService.AdvancedMessage2'

    __reflection: str
    story: str